src/lib/pubkey/kyber/kyber_common/kyber.cpp

```diff
@@ -117,6 +117,8 @@ class KyberConstants
                 return "Kyber-1024-r3";
             case KyberMode::Kyber1024_90s:
                 return "Kyber-1024-90s-r3";
+            default:
+                BOTAN_ASSERT(false, "unreachable code");
```

Comment:
Do you think it'd be worthwhile to introduce an explicit `BOTAN_UNREACHABLE`? Either simply as an alias of the above or via `__builtin_unreachable` and friends.

Reply:
Sounds good. I can do this in a new PR to master (or you can add one in this PR, either way is fine with me)

Reply:
we'll add it in a cherry-pickalbe commit on this branch

Reply:
it's b0025e10381afde1e1d9823c3f08886235a7bb85, in case it's needed before we merge this